pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/internal/PulsarAdminBuilderImpl.java

```diff
@@ -93,4 +93,16 @@ public PulsarAdminBuilder enableTlsHostnameVerification(boolean enableTlsHostnam
         conf.setTlsHostnameVerificationEnable(enableTlsHostnameVerification);
         return this;
     }
+
+    @Override
+    public PulsarAdminBuilder setConnectionTimeOutSeconds(int connectionTimeOutSeconds) {
+        conf.setConnectTimeOutSeconds(connectionTimeOutSeconds);
```

Comment:
Instead of adding field in `ClientConfigurationData` that are only used for the admin client we can just add them in `PulsarAdminBuilderImpl`

Reply:
we can add it there
```
public class PulsarAdminBuilderImpl implements PulsarAdminBuilder {

    private long connectTimeOutSeconds = 60;
    private long readTimeOutSeconds = 60;
    protected final ClientConfigurationData conf;
```
but all the constructors for PulsarAdmin will have to be overridden to take these two new integer values. I am not sure we wan to do that.

Reply:
it's done